### PR TITLE
Adding a landing page for Flavors.

### PR DESCRIPTION
**Pull request by sfshaza2**
As requested by @dnfield. Please review!

### CODE REVIEW

File: src/docs/deployment/flavors.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Creating flavors for Flutter
+description: How to create build flavors specific to different devices.
+---
+
+Do you need to set up product flavors for different devices?
```
Comment (dnfield) [inline src/docs/deployment/flavors.md]: `Do you need to set up product flavors for different development environments or release types?` may make more sense (flavors aren't necessarily device specific).

File: src/docs/deployment/flavors.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Creating flavors for Flutter
+description: How to create build flavors specific to different devices.
```
Comment (chalin) [inline src/docs/deployment/flavors.md]: Add a `short-title`? E.g.,

`short-title: Flavors`

**Comment by sfshaza2** [comment]
Thanks for that quick review, @dnfield! Your feedback is incorporate4d.

**Comment by dnfield** [comment]
No problem!

/cc @xster

**Comment by xster** [comment]
LGTM

**Pull request merged**